src/uikit.cs

```diff
@@ -6712,6 +6812,262 @@ interface UIScreenEdgePanGestureRecognizer {
+		[Notification]
+		NSString WillDeactivateNotification { get; }
+
+		[iOS (13,0)]
+		[Field ("UISceneWillEnterForegroundNotification")]
+		[Notification]
+		NSString WillEnterForegroundNotification { get; }
+
+		[iOS (13,0)]
+		[Field ("UISceneDidEnterBackgroundNotification")]
+		[Notification]
+		NSString DidEnterBackgroundNotification { get; }
+	}
+
+	interface IUISceneDelegate { }
+
+	[iOS (13, 0)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject))]
+	interface UISceneDelegate {
+
+		[Export ("scene:willConnectToSession:options:")]
+		void WillConnect (UIScene scene, UISceneSession session, UISceneConnectionOptions connectionOptions);
+
+		[Export ("sceneDidDisconnect:")]
+		void DidDisconnect (UIScene scene);
+
+		[Export ("sceneDidBecomeActive:")]
+		void DidBecomeActive (UIScene scene);
+
+		[Export ("sceneWillResignActive:")]
+		void WillResignActive (UIScene scene);
+
+		[Export ("sceneWillEnterForeground:")]
+		void WillEnterForeground (UIScene scene);
+
+		[Export ("sceneDidEnterBackground:")]
+		void DidEnterBackground (UIScene scene);
+
+		[Export ("scene:openURLContexts:")]
+		void OpenUrlContexts (UIScene scene, NSSet<UIOpenUrlContext> urlContexts);
+
+		[Export ("stateRestorationActivityForScene:")]
+		[return: NullAllowed]
+		NSUserActivity GetStateRestorationActivity (UIScene scene);
+
+		[Export ("scene:willContinueUserActivityWithType:")]
+		void WillContinueUserActivity (UIScene scene, string userActivityType);
+
+		[Export ("scene:continueUserActivity:")]
+		void ContinueUserActivity (UIScene scene, NSUserActivity userActivity);
+
+		[Export ("scene:didFailToContinueUserActivityWithType:error:")]
+		void DidFailToContinueUserActivity (UIScene scene, string userActivityType, NSError error);
+
+		[Export ("scene:didUpdateUserActivity:")]
+		void DidUpdateUserActivity (UIScene scene, NSUserActivity userActivity);
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UISceneActivationConditions : NSSecureCoding {
+
+		[Export ("init")]
+		[DesignatedInitializer]
+		IntPtr Constructor ();
+
+		[Export ("canActivateForTargetContentIdentifierPredicate", ArgumentSemantic.Copy)]
+		NSPredicate CanActivateForTargetContentIdentifierPredicate { get; set; }
+
+		[Export ("prefersToActivateForTargetContentIdentifierPredicate", ArgumentSemantic.Copy)]
+		NSPredicate PrefersToActivateForTargetContentIdentifierPredicate { get; set; }
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof (NSObject))]
+	interface UISceneActivationRequestOptions {
+
+		[NullAllowed, Export ("requestingScene", ArgumentSemantic.Strong)]
+		UIScene RequestingScene { get; set; }
+	}
+
+	[iOS (13,0), NoWatch]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UISceneConfiguration : NSCopying, NSSecureCoding {
+
+		[Static]
+		[Export ("configurationWithName:sessionRole:")]
+		UISceneConfiguration FromName ([NullAllowed] string name, [BindAs (typeof (UISceneSessionRole))] NSString sessionRole);
```

Comment:
No verb in method name.

Reply:
Same justification https://github.com/xamarin/xamarin-macios/pull/6392/files#r296216711